administrator-manual/en/fail2ban.rst

```diff
@@ -0,0 +1,160 @@
+========
+Fail2ban
+========
+
+Fail2ban scans log files (e.g. /var/log/apache/error_log) and bans IPs that show the malicious signs – too many password failures, seeking for exploits, etc. Generally Fail2Ban is then used to update firewall rules to reject the IP addresses for a specified amount of time, although any arbitrary other action (e.g. sending an email) could also be configured. Out of the box Fail2Ban comes with filters for various services (apache, courier, ssh, etc).
```

Comment:
/var/log/apache/error_log should be marked as :file:`/var/log/apache/error_log`
See https://github.com/NethServer/docs#editing-guidelines

Also use the correct case for:
Apache, Courier, SSH